src/contentmanager.cpp

```diff
@@ -780,7 +784,9 @@ void ContentManager::updateRemoteLibrary(const QString& content) {
         manager.readOpds(content.toStdString(), catalogUrl.toStdString());
         emit(this->booksChanged());
         emit(this->pendingRequest(false));
-    });
+    };
+    // wrap the lambda in LambdaRunnable for Qt 5.12 and later compat
+    QThreadPool::globalInstance()->start(new portutils::LambdaRunnable(f));
```

Comment:
Memory leak (since `LambdaRunnable` doesn't enable [auto-deletion](https://doc.qt.io/qt-5/qrunnable.html#setAutoDelete))

Reply:
And use `portutils::runConcurrently()` (as suggested in the other comment) here too

Reply:
There shouldn't be a memory leak because autoDelete is [enabled by default](https://doc.qt.io/qt-5/qrunnable.html#QRunnable).